src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -210,10 +223,12 @@ public function getFilters()
                 'sonata_urlsafeid',
                 [$this, 'getUrlSafeIdentifier']
             ),
+            //NEXT_MAJOR remove this filter
             new TwigFilter(
```

Comment:
Since this filter is added in `XEditableExtension`, isn't there a conflict, having to filter with the same name ?
We could remove the declaration from here (and accept the little BC). WDYT @franmomu ?